src/actions/google/drive/sheets/google_sheets.ts

```diff
@@ -161,7 +163,7 @@ export class GoogleSheetsAction extends GoogleDriveAction {
         const spreadsheetId = files.data.files[0].id!
 
         const sheets = await this.retriableSpreadsheetGet(spreadsheetId, sheet,
-            0, request.webhookId!).catch((e: any) => {
+            1, request.webhookId!).catch((e: any) => {
```

Comment:
magic number - can we put this under a variable so we have context on what it means?  (I assume it's retry count here??)

Reply:
also are we just starting at retry 1 now?

Reply:
It is retry count. This one is overkill to put into a const IMO

Reply:
I think it's worth naming this for clarity